etna/transforms/math/statistics.py

```diff
@@ -198,7 +198,9 @@ def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
         result: pd.DataFrame
             dataframe with results
         """
-        window = self.window if self.window != -1 else len(df)
+        window = self.window
+        if self.window == -1:
```

Comment:
How was it working without this fix?

Reply:
It wan't working, we have never used such combination